pebble_tool/commands/emucontrol.py

```diff
@@ -189,9 +190,8 @@ def __call__(self, args):
 
         try:
             max_angle_radians = 0x10000
-            max_angle_degrees = 360
-            offset = 180
-            heading = (args.heading * max_angle_radians + offset) / max_angle_degrees
+            max_angle_degrees = 360.0
```

Comment:
Yes it was, did not realize the future had these things, thanks for the pointer!